projects/train/train/data/waveforms/generator/cbc.py

```diff
@@ -0,0 +1,131 @@
+import math
+from functools import partial
+from typing import Any, Callable, Dict
+
+import torch
+from train.data.waveforms.generator.generator import WaveformGenerator
+
+
+class CBCGenerator(WaveformGenerator):
```

Comment:
Let's add FrequencyDomain in the name, since this will work when the callable returns h(f)